### DISCUSSION TITLE
You-Dont-Need-Lodash-Underscore support?

### DISCUSSION THREAD

**Discussion by kkimdev** [biomejs/biome#3363] [Ideas]
https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore is a super popular (18k Github stars) ESLint plugin that suggests better alternative to lodash usages.  It has been very helpful to our team as well.  It would be great for biome to support this.

I don't know if it's appropriate to implement this as a first party rules, because it's library specific though. (Does biome support plugin rules yet?)

**Comment by arendjr** [comment]
Plugins are still work-in-progress, although we have other framework-specific rules. Please use this thread for requesting rules based on ESLint plugins: https://github.com/biomejs/biome/discussions/3
